partner_contact_age_range/models/res_partner.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2019-Today: Druidoo (<https://www.druidoo.io>)
```

Comment:
get rid of the `(C)` here as well and replace Today:  with 2020

Reply:
@NL66278 done